src/flavors/__tests__/fixtures.ts

```diff
@@ -1,6 +1,8 @@
-import { Constructor, Flavor } from '../../interfaces';
```

Comment:
The `Constructor` type was fine, but was also redundant with `Type`. Eliminating our own type definition reduces the surface area of the non-options interfaces, which I think is good.